bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -794,6 +814,11 @@ lnd/clm2/surfdata_map/surfdata_ne120np4_78pfts_CMIP6_simyr1850_c170824.nc</fsurd
 <flanduse_timeseries hgrid="1x1_numaIA"    sim_year_range="1850-2000"
  use_crop=".true."   >lnd/clm2/surfdata_map/landuse.timeseries_1x1_numaIA_hist_78pfts_CMIP6_simyr1850-2015_c170917.nc</flanduse_timeseries>
 
+<flanduse_timeseries hgrid="C24"      sim_year_range="1850-2000"
+ use_crop=".true."  >lnd/clm2/surfdata_map/release-clm5.0.30/landuse.timeseries_C24_SSP5-8.5_78pfts_CMIP6_simyr1850-2100_c200317.nc</flanduse_timeseries>
```

Comment:
use_crop setting should be removed here, so that it can be used for both use_crop=".true. and false. Same for C96 as well. You should also add a compset that can test it, and make sure it works.

Reply:
@jtruesdal I think this might be my last change to resolve.

Reply:
- [ ] Resolved

Reply:
OK I got rid of the use_crop settings for C24 and C96 in the defaults file. Thanks for the reminder.  

I will need help setting up the compset.  I have no idea how the nomenclature of the alias IHistClm50SpGs should map into the long name. For instance how should it differ from IHistClm50SpG.

I'm happy to run a test but will need help setting it up and identifying what to look at to make sure the results are valid.  Or is it just a smoke test to make sure it runs?

Reply:
Now that the cime version is updated the compsets aren't as picky anymore. We had to remove the Glacier model previously, because there weren't mapping files for it. Now, there is, so a general compset should work.